pkg/codegen/schema.go

```diff
@@ -65,7 +86,7 @@ func (p Property) GoFieldName() string {
 
 func (p Property) GoTypeDef() string {
 	typeDef := p.Schema.TypeDecl()
-	if !p.Schema.SkipOptionalPointer && !p.Required {
```

Comment:
This will cause regressions. What if something is required but FieldNullable is at its default zerovalue of false? See my note about having a smarter Required() function. I don't like having two roughly synonymous flags.